src/pudl/extract/xbrl.py

```diff
@@ -158,10 +160,10 @@ def convert_form(
     form: XbrlFormNumber,
     datastore: FercXbrlDatastore,
     sqlite_engine: sa.engine.Engine,
-    output_path: Path = None,
+    output_path: Path,
```

Comment:
Changed the output type to `None` since that seems to be what it truly is; and it looks like we don't do any null handling on `output_path` so we should make sure it's only knowingly passed a `Path`.